tests/consensus_tests/test_routing_token.py

```
@@ -0,0 +1,158 @@
+def upsert_marker(peer_uri, marker):
+    """Upsert the single test point with the given `marker` payload on all replicas."""
+    r = upsert_points(
+        peer_uri,
+        [{"id": POINT_ID, "vector": VECTOR, "payload": {"marker": marker}}],
+        wait="true",
+        ordering="strong",
+    )
+    assert_http_ok(r)
+
+
+def retrieve_marker(peer_uri, headers={}):
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)